modules/example-apache-httpd-configuration.adoc

```diff
@@ -57,6 +59,20 @@ echo '{"error": "Login failure"}'
 exit 0
 ----
 
+.Results from Example
+----
+$ oc login -u basicauth-xxia1 -p redhat
```

Comment:
Line 64 uses the password, but your doc does not include how it is define somewhere else. This will confuse readers.

Reply:
Actually, you said `"This section provides examples with an httpd server configured as ... pod in cluster"`, readers will be confused where it `provides` the "examples" that do configure it as ... pod in cluster? The step examples that do configure it as ... pod in cluster using the oc commands demonstrated are missed.

Reply:
I added the passwords and OC commands that were missing. Please let me know if any changes are needed. Again, thank you so much for your help and patience!